llms/mlx_lm/utils.py

```diff
@@ -185,7 +237,7 @@ def generate(
             print("No tokens generated for this prompt")
             return
         prompt_tps = prompt.size / prompt_time
```

Comment:
Somehow I am getting an error on `prompt_tps = prompt.size / prompt_time`. I suspect it is because you changed the `prompt` to `prompt_tokens`, so it throws the error "str" doesn't have a size property.